fiftyone/core/camera.py

```
@@ -10,9 +10,9 @@
 |
 """
 
-import warnings
 from abc import ABC, abstractmethod
-from typing import Optional, Union
+from typing import Optional, Tuple, Union
```

Ruff (0.14.10)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)